dluhc-component-library/src/components/timetableForm/TimetableForm.tsx

```diff
@@ -86,7 +92,9 @@ const TimetableForm = () => {
     setData({ ...data, [key]: value });
   };
 
-  const Page = renderPage(currentPage, data, handleValueChange);
+  const Page = renderPage(currentPage, data, handleValueChange, () =>
```

Comment:
Sorry this was my suggestion, but I think it would be better if we moved the `handleContinueClicked` function on line 97 to be above this call to `renderPage`. Then we can just pass `handleContinueClicked` in like this: `const Page = renderPage(currentPage, data, handleValueChange, handleContinueClicked)` rather than having to redefine a new function each time.